src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-switcher.html

```diff
@@ -2,10 +2,10 @@
 {# As the theme switcher will only work when JavaScript is enabled, we add it through JavaScript. #}
 <script>
 document.write(`
-  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="{{ _('light/dark') }}" aria-label="{{ _('light/dark') }}" data-bs-placement="bottom" data-bs-toggle="tooltip">
-    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
-    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
-    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
+  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="{{ _('Color mode') }}" aria-label="{{ _('light/dark') }}" data-bs-placement="bottom" data-bs-toggle="tooltip">
```

Comment:
We need to provide the same label twice: once for [Bootstrap popovers](https://getbootstrap.com/docs/5.2/components/popovers/) (aka tooltips) and once for the accessible name (for assistive tech). Normally, a `title` attribute provided on a button would be picked up by the browser as an accessible name; however, the popover code from Bootstrap removes the attribute at runtime and stores it under a data attribute, `data-bs-title`, on the button element. To help avoid this confusion, I think we should stop using the `title` attribute in favor of `data-bs-title` since that's where it will eventually go anyway, and it makes it clearer that the value is Bootstrap (bs)-related.

```suggestion
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" aria-label="{{ _('Color mode') }}" data-bs-title="{{ _('Color mode') }}"  data-bs-placement="bottom" data-bs-toggle="tooltip">
```

Might be worth adding a comment about why the same translatable string is provided in two different places.